static/app/views/performance/transactionSummary/transactionEvents/eventsTable.tsx

```diff
@@ -322,7 +322,13 @@ class EventsTable extends Component<Props, State> {
         >
           {({pageLinks, isLoading, tableData}) => {
             const parsedPageLinks = parseLinkHeader(pageLinks);
-            const currentEvent = parsedPageLinks?.next?.cursor.split(':')[1] ?? 0;
+            let currentEvent = parsedPageLinks?.next?.cursor.split(':')[1] ?? 0;
+            if (
+              parseInt(currentEvent.replace(',', ''), 10) >
+              parseInt(totalEventCount.replace(',', ''), 10)
+            ) {
+              currentEvent = totalEventCount;
```

Comment:
I'm unsure of the data model here, but what happens if the user is on the last page? Say there are 54 events, and they're on page 2. Will `totalEventCount` be 4 or 54?

Reply:
It should be 54,
I was trying to be consistent with the issues page
https://sentry.io/organizations/sentry/issues/?cursor=1657125760000%3A0%3A0&page=30&query=is%3Aunresolved&statsPeriod=14d